src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/NoWhitespaceBeforeCheck.java

```diff
@@ -61,6 +61,19 @@
  * <pre>
  * &lt;module name=&quot;NoWhitespaceBefore&quot;/&gt;
  * </pre>
+ * <p>Example:</p>
+ * <pre>
+ * int c = a ++; // violation: '++' is preceded with whitespace
```

Comment:
`violation:` -> `violation, ` here and in all examples below
It is common to separate reason with comma